jena-text/pom.xml

```diff
@@ -112,11 +141,73 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <includes>
-            <include>**/TS_*.java</include>
-          </includes>
+            <!-- Skip the default running of this plug-in (or everything is run twice...see below) -->
+            <skip>true</skip>
+            <!--<excludedGroups>org.apache.jena.query.text.IntegrationTest</excludedGroups>-->
         </configuration>
+          <executions>
+              <execution>
+                  <id>unit-tests</id>
+                  <phase>test</phase>
+                  <goals>
+                      <goal>test</goal>
+                  </goals>
+                  <configuration>
+                      <skip>false</skip>
+                      <includes>
+                          <include>**/TS_*.java</include>
+                      </includes>
+                      <excludes>
+                          <exclude>**/*IT.java</exclude>
+                      </excludes>
+                  </configuration>
+              </execution>
+              <execution>
+                  <id>integration-tests</id>
+                  <phase>integration-test</phase>
+                  <goals>
+                      <goal>test</goal>
+                  </goals>
+                  <configuration>
+                      <skip>false</skip>
+                      <includes>
+                          <include>**/*IT.java</include>
+                      </includes>
+                  </configuration>
+              </execution>
+          </executions>
       </plugin>
+        <plugin>
+            <groupId>com.github.alexcojocaru</groupId>
+            <artifactId>elasticsearch-maven-plugin</artifactId>
+            <!-- REPLACE THE FOLLOWING WITH THE PLUGIN VERSION YOU NEED -->
+            <version>5.2</version>
+            <configuration>
+                <clusterName>elasticsearch</clusterName>
+                <tcpPort>9300</tcpPort>
```

Comment:
These are the default ES ports. In case there is already a running ES instance, the ports will clash. This is quite likely for someone who intends to use jena-text with ES. I suggest switching to e.g. ports 9500 and 9400